README.md

```diff
@@ -19,6 +19,8 @@ agent_zone=<name of the zone for the agent; default: nginx-tracing-demo>
 
 ## Build & Launch
 
+If you wish to test an NGINX tracing module built locally on your host, you can copy it into `nginx/ngx_http_opentracing_module.so` and uncomment the volumne mounting in the `nginx` service in the `docker-compose.yml` file.
+
```

Comment:
This whole repo is public and customer facing. This stuff is closed source. Customers cannot build their own modules. They can only receive some from us.

I'd like to drop this whole paragraph here. Let's keep it undocumented for not confusing customers.

```suggestion
```

Reply:
Removed, thanks